src/App.test.js

```diff
@@ -29,7 +29,8 @@ describe('App', () => {
       vistedParks: [],
       currentParksToShow: [],
       bucketListParks: [],
-      pageStatus: 'landing'
+      pageStatus: 'landing',
+      // need to check for randomImageClass - not sure how to test a random number in state.
```

Comment:
I'm not entirely sure the best way either. In a recent project though we tested that the number being returned from out random generator method was within the range we expected. Maybe we could try something like that for testing here?